template/tasks.py

```diff
@@ -224,6 +224,13 @@ def run_docs(ctx):
     ctx.run("poetry run mkdocs serve")
 
 
+@invoke.task
+def install(ctx, install_playwright: bool = False):
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** Please include a doc-block and a call to _title`

If you look at the other tasks in this file you can see that all have a short doc-block. Invoke uses these for the help test is present when you list the invoke tasks with `inv -l`.

You can also see that each of the tasks starts with a call to the `_install` function that just prints out a formatted title to make it clear what task is executing. This is more important for the tasks like `check` that run several other checks. But I think the consistency is nice.

Reply:
Updated in [28db98c](https://github.com/ackama/django-template/pull/46/commits/28db98cc5520755da41e5662dab813a876d33d2c)